### PR TITLE
creating get backup folder request to fetch the sql server default ba…

### PR DESCRIPTION
**Pull request by ssreerama**
…ckup folder info

This PR adds a service request to get the database backup folder details from ADS code that can be used in backup and restore tasks. STS changes are here: https://github.com/microsoft/sqltoolsservice/pull/2274

### CODE REVIEW

File: extensions/mssql/src/contracts.ts [hunk 1]
```diff
@@ -1684,10 +1684,18 @@ export interface GetDataFolderRequestParams {
 	connectionUri: string;
 }
 
+export interface GetBackupFolderRequestParams {
```
Comment (corivera) [inline extensions/mssql/src/contracts.ts:1687]: Small nit, but could you move this interface to just above GetBackupFolderRequest to keep the calls organized?

**Comment by corivera** [comment]
Has the STS version been bumped yet?

**Review by corivera — APPROVED** [review]
Looks good pending my other comments.

File: extensions/mssql/src/contracts.ts (hunk 1 above)
Comment (ssreerama) [reply extensions/mssql/src/contracts.ts]: Moved!

**Comment by ssreerama** [comment]
> Has the STS version been bumped yet?

Yes, Kim has already raised a PR and got merged! Thanks

**Pull request merged**